CMakeLists.txt

```
@@ -0,0 +1,36 @@
+################################################################################
+#    Copyright (C) 2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+cmake_policy(VERSION 3.14...3.20)
+
+project(FairCMakeModules VERSION 1.0 LANGUAGES NONE)
+
+include(CMakePackageConfigHelpers)
+
+set(PACKAGE_INSTALL_DESTINATION
+  share/cmake/${PROJECT_NAME}-${PROJECT_VERSION}
```

```suggestion
  ${CMAKE_INSTALL_DATAROOTDIR}/cmake/${PROJECT_NAME}-${PROJECT_VERSION}
```
from https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html. It is probably less risky to hard-code it in this case, but it is a very subtle aspect and I would rather have people copy&paste the configurable `share`.

---

I have a small problem with making that part being configurable.
The final directory needs to be discoverable by cmake. And cmake has a fixed list of subdirectories it searches.
See: https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure

---

Okay, thinking again about it:
If the user overrides `share`, they hopefully know, what they're doing.
For example GNUInstallDirs itself sets CMAKE_INSTALL_DATAROOTDIR to something like `usr/share` if one gives `CMAKE_INSTALL_PREFIX=/`.

So: Done.

---

> I have a small problem with making that part being configurable.
> The final directory needs to be discoverable by cmake. And cmake has a fixed list of subdirectories it searches.
> See: https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure

Ya, that is what I meant with "It is probably less risky to hard-code it in this case, but it is a very subtle aspect". And I would expect the find_package lookup to adapt pretty fast, if there was ever a mismatch. I doubt, the intention of looking in `share` is meant in a way that they want to look in exactly only `share` even though on a particular (future) system the policy is to name `share` something else^^